class.jetpack-gutenberg.php

```diff
@@ -151,18 +178,39 @@ public static function register( $slug, $args, $availability ) {
 		}
 	}
 
+	/**
+	 * Set a (non-block) extension as available
+	 *
+	 * @param string $slug Slug of the extension.
+	 */
+	public static function set_extension_available( $slug ) {
+		self::$availability[ self::remove_extension_prefix( $slug ) ] = true;
+	}
+
+	/**
+	 * Set the reason why an extension (block or plugin) is unavailable
+	 *
+	 * @param string $slug Slug of the extension.
+	 * @param string $reason A string representation of why the extension is unavailable.
+	 */
+	public static function set_extension_unavailable( $slug, $reason ) {
+		self::$availability[ self::remove_extension_prefix( $slug ) ] = $reason;
+	}
+
 	/**
 	 * Set the reason why an extension (block or plugin) is unavailable
 	 *
+	 * @deprecated 7.1.0 Use set_extension_unavailable() instead
+	 *
 	 * @param string $slug Slug of the extension.
 	 * @param string $reason A string representation of why the extension is unavailable.
 	 */
 	public static function set_extension_unavailability_reason( $slug, $reason ) {
-		self::$availability[ $slug ] = $reason;
+		self::set_extension_unavailable( $slug, $reason );
 	}
 
 	/**
-	 * Set up a whitelist of allowed block editor extensions
+	 * Set up a whitelist of allowed block editor extension cm[r[ryps
```

Comment:
what are `cm[r[ryps`?

Reply:
Thermos on keyboard when boarding an airplane without the screen locked :roll_eyes:

Reply:
7bf975d